backup-tools/backup-restore/app.go

```diff
@@ -71,12 +76,12 @@ func startRestoreInit(cc bool) error {
 		bufio.NewReader(os.Stdin).ReadBytes('\n')
 	}
 	if os.Getenv("BACKUP_PGSQL_FULL") != "" {
-		backupType = "pgsql"
+		utils.BackupType = "pgsql"
```

Comment:
If there is no support for restore etcd, why is this than in this PR? Do I oversee something